core-web/libs/sdk/client/src/lib/sdk-js-client.ts

```diff
@@ -145,100 +157,101 @@ export class DotCmsClient {
         }
 
         this.config = config;
+
+        this.requestOptions = {
+            ...this.config.requestOptions,
+            headers: {
+                Authorization: `Bearer ${this.config.authToken}`,
+                ...this.config.requestOptions?.headers
+            }
+        };
     }
 
-    /**
-     * `getPage` is an asynchronous method of the `DotCmsClient` class that retrieves all the elements of any Page in your dotCMS system in JSON format.
-     * It takes a `PageApiOptions` object as a parameter and returns a Promise that resolves to the response from the DotCMS API.
-     *
-     * The Page API enables you to retrieve all the elements of any Page in your dotCMS system.
-     * The elements may be retrieved in JSON format.
-     *
-     * @link https://www.dotcms.com/docs/latest/page-rest-api-layout-as-a-service-laas
-     * @method getPage
-     * @async
-     * @param {PageApiOptions} options - The options for the Page API call.
-     * @returns {Promise<unknown>} - A Promise that resolves to the response from the DotCMS API.
-     * @throws {Error} - Throws an error if the options are not valid.
-     */
-    async getPage(options: PageApiOptions): Promise<unknown> {
-        this.validatePageOptions(options);
+    page = {
+        /**
+         * `page.get` is an asynchronous method of the `DotCmsClient` class that retrieves all the elements of any Page in your dotCMS system in JSON format.
+         * It takes a `PageApiOptions` object as a parameter and returns a Promise that resolves to the response from the DotCMS API.
+         *
+         * The Page API enables you to retrieve all the elements of any Page in your dotCMS system.
+         * The elements may be retrieved in JSON format.
+         *
+         * @link https://www.dotcms.com/docs/latest/page-rest-api-layout-as-a-service-laas
+         * @async
+         * @param {PageApiOptions} options - The options for the Page API call.
+         * @returns {Promise<unknown>} - A Promise that resolves to the response from the DotCMS API.
+         * @throws {Error} - Throws an error if the options are not valid.
+         */
+        get: async (options: PageApiOptions): Promise<unknown> => {
```

Comment:
I know this should he a class, but we can work on it later.

Reply:
We can write the get method for the client and just instance a new client with the url for the page API or nav API